src/Compilers/VisualBasic/Portable/Compilation/SyntaxTreeSemanticModel.vb

```diff
@@ -864,7 +864,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' </summary>
         Private Function CheckSymbolLocationsAgainstSyntax(symbol As NamedTypeSymbol, nodeToCheck As VisualBasicSyntaxNode) As NamedTypeSymbol
             For Each location In symbol.Locations
-                If location.SourceTree Is Me.SyntaxTree AndAlso nodeToCheck.Span.Contains(location.SourceSpan.Start) Then
+                If location.SourceTree Is Me.SyntaxTree AndAlso nodeToCheck.Span.Contains(location.SourceSpan) Then
```

Comment:
I think teh right place would probably be: Open\src\Compilers\VisualBasic\Test\Semantic\Compilation\SemanticModelGetDeclaredSymbolAPITests.vb

Since you're doing this to affect how GetDeclaredSymbol works right?